cluster/gce/gci/configure-helper.sh

```diff
@@ -594,6 +594,7 @@ function create-master-auth {
   if [[ -n "${NODE_PROBLEM_DETECTOR_TOKEN:-}" ]]; then
     append_or_replace_prefixed_line "${known_tokens_csv}" "${NODE_PROBLEM_DETECTOR_TOKEN},"   "system:node-problem-detector,uid:node-problem-detector"
   fi
+  append_or_replace_prefixed_line "${known_tokens_csv}" "${ADDON_MANAGER_TOKEN},"   "system:addon-manager,uid:addon-manager"
```

Comment:
Probably check if `${ADDON_MANAGER_TOKEN}` is set like the others?

Reply:
Does this need to match the `system:addon-manager-role` role we create in `create-apply-addon-manager-rbac()`?

Reply:
1. Added the check.
2. In this case it needs to be the user which is bound to that role.